Makefile

```diff
@@ -0,0 +1,40 @@
+.PHONY: all
+all: build check
+
+.PHONY: check
+check: fmt vet lint ## Check your code
+
+.PHONY: revendor
+revendor:
+	rm glide.lock
+	glide install --strip-vendor
+	glide-vc --use-lock-file --no-tests --only-code
+
+
+.PHONY: bin/terraform
+bin/terraform:
+	hack/get-terraform.sh
+
+.PHONY:  bin/openshift-install ## build installer binary
+bin/openshift-install:
+	hack/build.sh
+
+.PHONY: test
+test: # Run unit test
```

Comment:
This single `#` is insufficient for your current regexp, so this isn't showing up in `help`.